Opensearch exporter fails with 400 on AWS Opensearch service

**Describe the bug**

When connecting the zeebe opensearch exporter with the [AWS Opensearch service](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/gsg.html) some requests (like removing all ISM policies from zeebe indices) fail with a HTTP 400 without further indication why in the response body.

**To Reproduce**

1. create an opensearch service on aws
2. connect the zeebe opensearch exporter to it

**Investigation**
Debugging the failing calls by executing them manually via curl revealed the cause is a missing "/" at the beginning of the path like [here](https://github.com/camunda/zeebe/blob/main/zeebe/exporters/opensearch-exporter/src/main/java/io/camunda/zeebe/exporter/opensearch/OpensearchClient.java#L272).

See the following curl example that tries the same request without and with the leading "/" using the `--request-target` param.

```
➜  camunda-platform git:(stable/8.4) ✗ curl --location --request-target "_plugins/_ism/remove/zeebe-record*" --request POST 'https://search-operate-XXX.eu-north-1.es.amazonaws.com'  -i \
--header 'Authorization: Basic ****'
HTTP/2 400
server: awselb/2.0
date: Mon, 06 May 2024 06:20:01 GMT
content-type: text/html
content-length: 122

<html>
<head><title>400 Bad Request</title></head>
<body>
<center><h1>400 Bad Request</h1></center>
</body>
</html>
➜  camunda-platform git:(stable/8.4) ✗ curl --location --request-target "/_plugins/_ism/remove/zeebe-record*" --request POST 'https://search-operate-XXX.eu-north-1.es.amazonaws.com'  -i \
--header 'Authorization: Basic ***'
HTTP/2 200
date: Mon, 06 May 2024 06:20:07 GMT
content-type: application/json; charset=UTF-8
content-length: 981
access-control-allow-origin: *

{"updated_indices":0,"failures":true,"failed_indices":[{"index_name":"zeebe-record_command-distribution_8.5.0_2024-05-03","index_uuid":"uUYcPuUOSES-xEwQNPKr0g","reason":"This index does not have a policy to remove"},{"index_name":"zeebe-record_job_8.5.0_2024-05-03","index_uuid":"BwP_sMgoQAKqzVLPX1mg_w","reason":"This index does not have a policy to remove"},{"index_name":"zeebe-record_process-instance_8.5.0_2024-05-03","index_uuid":"evL-SBEmS3OKmK5RqeaiYA","reason":"This index does not have a policy to remove"},{"index_name":"zeebe-record_variable_8.5.0_2024-05-03","index_uuid":"UBoLC6xMQL6xq5nv8cvwQg","reason":"This index does not have a policy to remove"},{"index_name":"zeebe-record_process_8.5.0_2024-05-03","index_uuid":"m9WaH1M0S86-ZgCx5FVEvw","reason":"This index does not have a policy to remove"},{"index_name":"zeebe-record_process-instance-creation_8.5.0_2024-05-03","index_uuid":"nvjZBDn8TOCQkKeGoJ1XmQ","reason":"This index does not have a policy to remove"}]}%
```


**Expected behavior**

Requests should succees like on a bare opensearch instance as used during integration testing.

**Log/Stacktrace**

<!-- If possible add the full stacktrace or Zeebe log which contains the issue. -->

<details><summary>Full Stacktrace</summary>
 <p>

```
024-05-03 10:45:12 2024-05-03 07:45:12.069 [Broker-0] [zb-fs-workers-0] [Exporter-4] WARN 
2024-05-03 10:45:12       io.camunda.zeebe.broker.exporter.opensearch - Error on exporting record with key 2251799813685249
2024-05-03 10:45:12 io.camunda.zeebe.exporter.opensearch.OpensearchExporterException: Failed to add policy to indices
2024-05-03 10:45:12     at io.camunda.zeebe.exporter.opensearch.OpensearchClient.bulkRemoveISMPolicyToAllZeebeIndices(OpensearchClient.java:275) ~[zeebe-opensearch-exporter-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.exporter.opensearch.OpensearchExporter.updateRetentionPolicyForExistingIndices(OpensearchExporter.java:345) ~[zeebe-opensearch-exporter-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.exporter.opensearch.OpensearchExporter.export(OpensearchExporter.java:96) ~[zeebe-opensearch-exporter-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.broker.exporter.stream.ExporterContainer.lambda$export$5(ExporterContainer.java:188) ~[zeebe-broker-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.util.jar.ThreadContextUtil.runCheckedWithClassLoader(ThreadContextUtil.java:58) ~[zeebe-util-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.util.jar.ThreadContextUtil.runWithClassLoader(ThreadContextUtil.java:34) ~[zeebe-util-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.broker.exporter.stream.ExporterContainer.export(ExporterContainer.java:187) ~[zeebe-broker-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.broker.exporter.stream.ExporterContainer.exportRecord(ExporterContainer.java:174) ~[zeebe-broker-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.broker.exporter.stream.ExporterDirector$RecordExporter.export(ExporterDirector.java:532) ~[zeebe-broker-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.scheduler.retry.BackOffRetryStrategy.run(BackOffRetryStrategy.java:51) ~[zeebe-scheduler-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.scheduler.ActorJob.invoke(ActorJob.java:94) [zeebe-scheduler-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.scheduler.ActorJob.execute(ActorJob.java:47) [zeebe-scheduler-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.scheduler.ActorTask.execute(ActorTask.java:122) [zeebe-scheduler-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.scheduler.ActorThread.executeCurrentTask(ActorThread.java:130) [zeebe-scheduler-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.scheduler.ActorThread.doWork(ActorThread.java:108) [zeebe-scheduler-8.4.6.jar:8.4.6]
2024-05-03 10:45:12     at io.camunda.zeebe.scheduler.ActorThread.run(ActorThread.java:227) [zeebe-scheduler-8.4.6.jar:8.4.6]
2024-05-03 10:45:12 Caused by: org.opensearch.client.ResponseException: method [POST], host [https://search-operate-opensearch-2-11-3nkp5dzczb7ntz7gekco3f34em.eu-north-1.es.amazonaws.com/], URI [_plugins/_ism/remove/zeebe-record*], status line [HTTP/1.1 400 Bad Request]
2024-05-03 10:45:12 <html>
2024-05-03 10:45:12 <head><title>400 Bad Request</title></head>
2024-05-03 10:45:12 <body>
2024-05-03 10:45:12 <center><h1>400 Bad Request</h1></center>
2024-05-03 10:45:12 </body>
2024-05-03 10:45:12 </html>
```

</p>
</details>

**Environment:**
- OS: <!-- [e.g. Linux] -->
- Zeebe Version: 8.4.x & 8.5.x
- Configuration:
```
ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_PASSWORD=***
ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_USERNAME=***
ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_BULK_SIZE=1
ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_INDEX_NUMBEROFREPLICAS=2
ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_INDEX_NUMBEROFSHARDS=1
ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_URL=https://search-XXX.eu-north-1.es.amazonaws.com/
ZEEBE_BROKER_EXPORTERS_OPENSEARCH_CLASSNAME=io.camunda.zeebe.exporter.opensearch.OpensearchExporter
```

--- a/zeebe/exporters/opensearch-exporter/src/main/java/io/camunda/zeebe/exporter/opensearch/OpensearchClient.java
+++ b/zeebe/exporters/opensearch-exporter/src/main/java/io/camunda/zeebe/exporter/opensearch/OpensearchClient.java
@@ -219,7 +219,7 @@ public class OpensearchClient implements AutoCloseable {
   Optional<GetIndexStateManagementPolicyResponse> getIndexStateManagementPolicy() {
     try {
       final var request =
-          new Request("GET", "_plugins/_ism/policies/" + configuration.retention.getPolicyName());
+          new Request("GET", "/_plugins/_ism/policies/" + configuration.retention.getPolicyName());
       return Optional.of(sendRequest(request, GetIndexStateManagementPolicyResponse.class));
     } catch (final IOException e) {
       return Optional.empty();
@@ -240,7 +240,7 @@ public class OpensearchClient implements AutoCloseable {
     try {
       final var request =
           new Request(
-              "DELETE", "_plugins/_ism/policies/" + configuration.retention.getPolicyName());
+              "DELETE", "/_plugins/_ism/policies/" + configuration.retention.getPolicyName());
 
       final var response = sendRequest(request, DeleteStateManagementPolicyResponse.class);
       return response.result().equals(DeleteStateManagementPolicyResponse.DELETED);
@@ -252,7 +252,7 @@ public class OpensearchClient implements AutoCloseable {
   private boolean putIndexStateManagementPolicy(final Map<String, String> queryParameters) {
     try {
       final var request =
-          new Request("PUT", "_plugins/_ism/policies/" + configuration.retention.getPolicyName());
+          new Request("PUT", "/_plugins/_ism/policies/" + configuration.retention.getPolicyName());
 
       queryParameters.forEach(request::addParameter);
 
@@ -269,7 +269,7 @@ public class OpensearchClient implements AutoCloseable {
   public boolean bulkAddISMPolicyToAllZeebeIndices() {
     try {
       final var request =
-          new Request("POST", "_plugins/_ism/add/" + configuration.index.prefix + "*");
+          new Request("POST", "/_plugins/_ism/add/" + configuration.index.prefix + "*");
       final var requestEntity = new AddPolicyRequest(configuration.retention.getPolicyName());
       request.setJsonEntity(MAPPER.writeValueAsString(requestEntity));
       final var response = sendRequest(request, IndexPolicyResponse.class);
@@ -282,7 +282,7 @@ public class OpensearchClient implements AutoCloseable {
   public boolean bulkRemoveISMPolicyToAllZeebeIndices() {
     try {
       final var request =
-          new Request("POST", "_plugins/_ism/remove/" + configuration.index.prefix + "*");
+          new Request("POST", "/_plugins/_ism/remove/" + configuration.index.prefix + "*");
       final var response = sendRequest(request, IndexPolicyResponse.class);
       return !response.failures();
     } catch (final IOException e) {

